security/KeePassXC/Portfile

```diff
@@ -50,16 +50,15 @@ if {${subport} eq ${name}} {
                         patch-old-mac.diff
 } else {
     # devel subport
-    github.setup        keepassxreboot keepassxc a3dc977e58470644f2acca77905285d44b22f2b8
+    github.setup        keepassxreboot keepassxc d3b28f86515df73194d1102253b739b51b1909f5
     set githash         [string range ${github.version} 0 6]
-    version             20211122+git${githash}
-    revision            1
+    version             20211123+git${githash}
```

Comment:
You might want to keep a "revision 0" there.

Reply:
Initially there was no revision at all and it seems the policy on this varies in time. Some prefer to have it removed when version changes others prefer to keep it with 0. So how to know what is expected in which situation?

Btw it is likely there will be a future revision soon anyway either with additional fixes (depending on whether other build issues will reveal), or once upstream merged the pending pr.

Shall I recommit with that single change or can it wait for the next PR ?

Reply:
The policy changed. Originally we did not leave "revision 0" in Portfiles but now we do. This is a consistent policy across the project.

It is fine to ask for multiple small pulls. In fact, that makes review easier. Regardless, do you wish for me to merge this now?

Reply:
Ok thanks for clarification.
Yes please, merge this.

Reply:
Could you put the "revision 0" in first? As soon as you push that I'll merge.

Reply:
Done. Thanks :)